### PR TITLE
backport: signup changes from www.app.decenterai.com

### PR DESCRIPTION
**Pull request by techyNonso**
## Related Issue(s)

Fixes #
<br>
For #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## To-Do

- [ ]

### CODE REVIEW

**Comment by techyNonso** [comment]
@Nasfame please review this as soon as possible

File: src/app/components/home/Hero.tsx [hunk 1]
```diff
@@ -107,7 +112,11 @@ const Hero = () => {
             Train ML/DL/ ANN/LLM models with ease.
           </p>
           <div className="flex space-x-9 md:space-x-16  md:mt-10">
-            <Link href="https://app.decenterai.com/signup">
+
+            <Link href="https://app.decenterai.com/explore">
```
Comment (Nasfame) [inline src/app/components/home/Hero.tsx:116]: just direct to https://app.decenterai.com/

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: src/app/components/home/Hero.tsx [hunk 2]
```diff
@@ -85,7 +87,10 @@ const Hero = () => {
           </div>
 
           <div className=" w-[50%] md:w-[20%] h-full flex items-center justify-end">
-            <Link href="https://app.decenterai.com/signup" className="hidden md:block">
+
+            <Link href="https://app.decenterai.com/explore" className="hidden md:block">
```
Comment (Nasfame) [inline src/app/components/home/Hero.tsx:91]: don't do a refactor to https://app.decenterai.com. no explore. We cannot do this all the time,

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: src/app/components/home/Hero.tsx (hunk 2 above)
Comment (Nasfame) [reply src/app/components/home/Hero.tsx:91]: Do a refactor to `https://app.decenterai.com/.` not /explore

Comment (Nasfame) [reply src/app/components/home/Hero.tsx:91]: @techyNonso

**Review by Nasfame — APPROVED** [review]
LGTM: @techyNonso u can merge if thats all the changes

**Pull request merged**